### DISCUSSION TITLE
Is progressr known to fail if multisession is used in Linux?

### DISCUSSION THREAD

**Discussion by iagomosqueira** [futureverse/progressr#161] [Q&A]
Progress is correctly reported when using `plan(multicore)` in Linux, but fails to work if `plan(multisession`) is called. `handlers('debug')` does not show any output.

Is this a known issue?

**Comment by iagomosqueira** [comment]
Apologies, some error in the code was making it fail with multisession

**Reply by scottkosty** [reply]
@iagomosqueira now you have my curiousity! What error was it? I'm curious why it worked with multicore but not multisession.

By the way, unless I'm mistaken I believe "multisession" is the recommended plan on Linux.
